### PR TITLE
do not destructure arguments in _

### PR DESCRIPTION
**Pull request by kurtharriger**
https://github.com/paldepind/union-type/issues/24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +118,7 @@ const advancePlayer = (move, player) =>
     Right: () => ({x: player.x + 1, y: player.y}),
     Down: () => ({x: player.x, y: player.y + 1}),
     Left: () => ({x: player.x - 1, y: player.y}),
+    _: () => (player),
```
Comment (davidchambers) [inline README.md]: Shall we drop the parens around `player`?

Comment (kurtharriger) [inline README.md]: I like that it is consistent with the lines above, but not strongly opinionated about it.

Comment (davidchambers) [inline README.md]: We'll let @paldepind cast the deciding vote. :)

Comment (kurtharriger) [inline README.md]: Changed my mind. Noticed that elsewhere in the docs the parens are not used, so if I really wanted to be consistent on this I should have changed it there too.

File: README.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ constructors. `case` can be used as a control structure for handling the
 different values a type can have:
 
 ```javascript
-var Move = Type({Up: [], Right: [], Down: [], Left: []});
+var Move = Type({Up: [], Right: [], Down: [], Left: [], Jump: [], Fire: [Number]});
```
Comment (paldepind) [inline README.md]: What about renaming `Move` to `Action`? Given these two additions that sounds like a better name to me.

**Comment by paldepind** [comment]
Thank you for the pull request! I've left a single nitpicking comment. Otherwise it looks great!

**Comment by kurtharriger** [comment]
Updated

**Comment by paldepind** [comment]
Thank you!

**Pull request merged**